### PR TITLE
Refactor overall HTML and CSS

### PR DESCRIPTION
**Pull request by yuriko627**
- Fix Exchange page (order book page) UI 
- Add order request modal

### CODE REVIEW

File: src/components/Layout.js [hunk 1]
```diff
@@ -26,6 +29,9 @@ const Layout = props => {
         <div className="layout">
           <div className="main">
             <MainDisplay>{props.children}</MainDisplay>
+            <div className="wallet-id-section">
+              Your wallet ID: 0x00000000000000000000000000000000
```
Comment (tkmct) [inline src/components/Layout.js]: Should this be address from client?

File: src/components/Layout.js [hunk 2]
```diff
@@ -55,6 +65,7 @@ const Layout = props => {
           ${isDepositModalOpen ? 'overflow: hidden;' : ''}
```
Comment (tkmct) [inline src/components/Layout.js]: you can merge these conditions into one condition.

File: src/colors.js [hunk 3]
```diff
@@ -1,5 +1,15 @@
+export const White = opacity => {
+  return `rgba(255, 255, 255, ${opacity} )`
+}
+export const Black = opacity => {
+  return `rgba(0, 0, 0, ${opacity} )`
+}
 export const BACKGROUND = '#1B191A'
+export const SECTION_BACKGROUND = White(0.08)
```
Comment (tkmct) [inline src/colors.js]: good!

File: src/components/Send.js [hunk 4]
```diff
@@ -30,18 +27,18 @@ const Send = props => {
   const transferredToken = props.transferredToken
   const tokenBalanceList = props.tokenBalanceList
 
-  useEffect(() => {
-    props.getBalance()
-    props.getETHtoUSD()
-  }, [])
+  // useEffect(() => {
```
Comment (tkmct) [inline src/components/Send.js]: if you don't use these lines anymore, please remove lines.
if you do, please add code comment

File: src/components/Send.js [hunk 5]
```diff
@@ -273,40 +265,48 @@ const Send = props => {
...
+        .amount-box {
           margin-left: 10px;
+          display: flex;
+          flex-direction: column;
+        }
+        .amount-input {
           height: 47px;
           width: 85px;
           padding: 4px;
           font-size: 36px;
```
Comment (tkmct) [inline src/components/Send.js]: what about make font-size constant variable as well?

File: src/components/TransactionHistory.js [hunk 6]
```diff
@@ -94,8 +94,7 @@ const TransactionHistory = props => {
           overflow-y: scroll;
           height: calc(100vh - 7.5rem);
         }
-        .transaction-hisotry-title {
-          color: rgba(255, 255, 255, 0.75);
+        .transaction-hisotry-t${White(0.75)};
```
Comment (tkmct) [inline src/components/TransactionHistory.js]: what is this? seems like an error.

File: src/store/appStatus.js [hunk 7]
```diff
@@ -35,6 +36,10 @@ export const checkClientInitialized = () => {
     if (client) {
       dispatch(setAppStatus(APP_STATUS.LOADED))
       dispatch(subscribeEvents())
+      dispatch(getBalance())
```
Comment (tkmct) [inline src/store/appStatus.js]: what about extracting these `get*` things into a function.

File: src/components/Layout.js (hunk 1 above)
Comment (yuriko627) [reply src/components/Layout.js]: you are right

File: src/components/Send.js (hunk 5 above)
Comment (yuriko627) [reply src/components/Send.js]: changed every font-size to constant variables.

File: src/components/TransactionHistory.js (hunk 6 above)
Comment (yuriko627) [reply src/components/TransactionHistory.js]: thanks for noticing!

File: src/components/Send.js (hunk 4 above)
Comment (yuriko627) [reply src/components/Send.js]: removed them.

File: src/store/appStatus.js (hunk 7 above)
Comment (yuriko627) [reply src/store/appStatus.js]: created initialGetters function.

File: src/components/AddressList/AddressListItem.js [hunk 8]
```diff
@@ -122,62 +101,59 @@ export default ({
...
       .name {
-        font-size: 18px;
-        font-weight: 800;
+        font-size: ${SMALLER};
+        font-weight: 500;
       }
       .address {
-        font-size: 14px;
-        font-weight: 800;
+        font-size: 12px;
```
Comment (tkmct) [inline src/components/AddressList/AddressListItem.js]: !!

File: src/components/AddressList/AddressListItem.js [hunk 9]
```diff
@@ -122,62 +101,59 @@ export default ({
...
+        height: 24px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        background-color: ${White(0.06)};
+        color: ${PRIMARY_BUTTON_TEXT};
         cursor: pointer;
-        font-size: 14px;
-        font-weight: 800;
+        font-size: 12px;
```
Comment (tkmct) [inline src/components/AddressList/AddressListItem.js]: !!

File: src/components/Layout.js [hunk 10]
```diff
@@ -69,26 +81,39 @@ const Layout = props => {
           flex-direction: column;
           width: 70%;
         }
+        .wallet-id-section {
+          height: 32px;
+          background-color: #2d2a2c;
+          font-size: 12px;
```
Comment (tkmct) [inline src/components/Layout.js]: !!

**Pull request merged**